templates/includes/card-group-promo--dark.html

```diff
@@ -1,33 +1,32 @@
 {% load wagtailimages_tags %}
 
-{% image page.teaser_image fill-314x253 as teaser_image_extra_small %}
-{% image page.teaser_image fill-340x274 as teaser_image_small %}
-{% image page.teaser_image fill-357x288 as teaser_image_medium %}
-{% image page.teaser_image fill-478x385 as teaser_image_large %}
-{% image page.teaser_image fill-513x413 as teaser_image_extra_large %}
+{% image page.teaser_image fill-446x273 as teaser_image_small %}
+{% image page.teaser_image fill-492x301 as teaser_image_medium %}
+{% image page.teaser_image fill-508x311 as teaser_image_large %}
+{% image page.teaser_image fill-626x383 as teaser_image_extra_large %}
 
 <li class="card-group-promo--dark">
     <h2 class="card-group-promo__heading">{{ heading }}</h2>
 
     <div class="card-group-promo__card">
         <div class="row">
-            <div class="col-md-6 col-lg-5 col-xl-4">
-                <a href="{{ page.url }}" tabindex="-1">
-                    <picture>
-                        <source media="(max-width: 576px)" srcset="{{ teaser_image_extra_large.url }}">
-                        <source media="(max-width: 768px)" srcset="{{ teaser_image_large.url }}">
-                        <source media="(max-width: 991px)" srcset="{{ teaser_image_extra_small.url }}">
+            <div class="col-md-12 col-lg-7 col-xl-6">
+                <a href="{{ page.url }}" tabindex="-1" class="card-group-promo__card-link">
+                    <picture class="mt-auto">
```

Comment:
Rather than using Bootstrap's `mt-auto` here, because this padding is a responsibility of the component I think the margin styles should be within the Sass for the component. I think it's fine to use something like `class="container mt-4"` as you have elsewhere because the relevant element is being controlled solely by Bootstrap. Hope that doesn't sound contradictory - happy to explain on a call if that would be helpful.

Reply:
Thanks @gtvj, looks like the class is not needed with the new styles anyway so I have removed it.